app/api/log/specs/errorLog.spec.js

```diff
@@ -35,7 +37,7 @@ describe('errorLog', () => {
       tenants.add({ name: 'tenant' });
 
       await tenants.run(async () => {
-        anErrorLog.error('a message', { shouldBeMultiTenantContext: true });
+        anErrorLog.error('a message');
```

Comment:
the describe for this test is outdated now after removing the flag right ?

Reply:
Yes, I have reverted the tests:
- default behavior now has no describe
- added describe for "unexpected" behavior